FiskalySDK/FiskalyRequestClient.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  FiskalyRequestClient.swift
+//  FiskalySDK
+//
+//  Created by Marcel Voß on 09.06.20.
+//  Copyright © 2020 fiskaly. All rights reserved.
+//
+
+import Foundation
+import FiskalyClient
+
+public protocol RequestClient {
+    func invoke(request: String) -> String
+}
+
+struct FiskalyRequestClient: RequestClient {
+    func invoke(request: String) -> String {
```

Comment:
This is essentially only about moving the previously called [FiskalyClientInvoke](https://github.com/fiskaly/fiskaly-sdk-swift/pull/16/files#diff-4539068c353ba920d717b8fe4fb234dfL4) function into a type conforming to the `RequestClient` protocol and allows an integrator to mock calls in their tests.